apps/sms/services/js/drafts.js

```diff
@@ -58,6 +58,7 @@
       draftIndex.set(draft.threadId, drafts);
 
       this.emit('saved', draft);
+      InterInstanceEventDispatcher.emit('drafts-saved', draft);
```

Comment:
__question:__ my initial idea was to fire event only when draft is actually stored and ```Drafts``` should have compare old and new draft sets and fire corresponding events. But I'm totally fine with your approach as my is more complex and we actually don't need it right now since ```DraftsService``` will replace this soon anyway - so no waste here :)

The only concern is batch deletion - could you please make sure that when we remove big chunk of conversations our memory numbers aren't going crazy (we issue postMessage and read IndexedDB)? I guess we need one active main and one activity instance to verify :)